lib/reducers/create-otp-reducer.js

```diff
@@ -991,6 +992,19 @@ function createOtpReducer(config) {
             }
           }
         })
+      case 'SET_POPUP_CONTENT':
+        const target = state.config?.popups?.targets?.[action.payload]
```

Comment:
Are there cases where `state.otp.config` could be null? (That's one of the first things loaded on startup)

Reply:
I was being better safe than sorry but you're right `config` is in the initializer so it should be fine